library/src/main/java/com/fidesmo/fdsm/FidesmoApiClient.java

```diff
@@ -119,7 +119,7 @@ public CloseableHttpResponse transmit(HttpRequestBase request) throws IOExceptio
 
         CloseableHttpResponse response = http.execute(request, context);
         int responsecode = response.getStatusLine().getStatusCode();
-        if (responsecode < 200 || responsecode > 299) {
+        if (responsecode < 200 || responsecode > 299 && responsecode != 424) {
```

Comment:
There's more statuses you can get on `/service/error` endpoint along with 424. Sec-server can return `401`, `204`, `502` etc. All of them should be ignored by the client for this endpoint as it doesn't tell anything to the user and we will hide those errors from the client in the future. But for other endpoints all those statuses matters and has to be taken into account